### PR TITLE
Badges

### PR DESCRIPTION
**Pull request by vanshitalavange**
Created Badge component with the following variants
 - icons with badge on left and right
 - avatar with badge
 - text with small and large badge
 - buttons with badge on left and right
 
image

### CODE REVIEW

File: stylesheets/components/badges.css [hunk 1]
```diff
@@ -0,0 +1,91 @@
...
+  top: 0;
+  right: 0;
+}
+.badge-counter-left {
+  top: 0;
+  left: 0;
+}
+
+.badge-counter { 
+
```
Comment (tanishq20) [inline stylesheets/components/badges.css]: Avoid giving these extra spaces.

File: stylesheets/components/badges.css [hunk 2]
```diff
@@ -0,0 +1,91 @@
...
+}
+.badge-round{
+    position: absolute;
+    z-index: 1;
+    border-radius: 50%;
+}
+/* icons with badge */
+.badge-icon {
+  color: var(--blue-semi-dark);
+  font-size: 2.5rem !important;
```
Comment (tanishq20) [inline stylesheets/components/badges.css:21]: You can try to do this without `!important`.
According to me `!important` should be the last option in the list to use.

**Review by tanishq20 — COMMENTED** [review]
Commented some small changes.
Rest everything is fine.

File: stylesheets/components/badges.css (hunk 2 above)
Comment (vanshitalavange) [reply stylesheets/components/badges.css:21]: Initially, I tried doing it without `!important` but it wasn't working, so I had to use `!important`

File: stylesheets/components/badges.css [hunk 3]
```diff
@@ -0,0 +1,91 @@
...
+/* buttons with badge */
+.badge-btn {
+  margin-top: 1rem;
+  margin-right: 0.5rem;
+  margin-left: 0.5rem;
+  padding: 0.5rem 1rem;
+  color: var(--white-color);
+  border-radius: 6px;
+}
+.badge-btn-primary {
```
Comment (vivekmehto) [inline stylesheets/components/badges.css]: you can use button classes to avoid extra code.

File: stylesheets/components/badges.css (hunk 1 above)
Comment (vanshitalavange) [reply stylesheets/components/badges.css]: Yup changes done, thanks

File: stylesheets/components/badges.css [hunk 4]
```diff
@@ -0,0 +1,91 @@
...
+  bottom: 0.2rem;
+}
+.badge-status-circle {
+  top: 0;
+  height: 100%;
+  width: 100%;
+  border: 2.5px solid var(--blue-semi-dark);
+}
+/* text with badge */
+.badge-text-content {
```
Comment (vivekmehto) [inline stylesheets/components/badges.css:57]: Your badge is hiding the text. Try changing top of badge.

File: stylesheets/components/badges.css (hunk 3 above)
Comment (vanshitalavange) [reply stylesheets/components/badges.css]: Thank you, Vivek for this suggestion! This really helped in minimizing the code

File: stylesheets/components/badges.css (hunk 4 above)
Comment (vanshitalavange) [reply stylesheets/components/badges.css:57]: Agreed! I've done the requested changes. Please check!
image

Comment (vivekmehto) [reply stylesheets/components/badges.css:57]: Looks perfect to merge now.

**Pull request merged**